lib/form/sfForm.class.php

```diff
@@ -316,7 +316,7 @@ public function isBound()
    */
   public function updateValues(array $values)
   {
-    $this->values = $values + $this->values;
+    $this->values = $values + (isset($this->values) ? $this->values : array());
```

Comment:
How about setting the initial value of $values to array()? https://github.com/FriendsOfSymfony1/symfony1/blob/c8cadb413101949316b0759510a1125580259647/lib/form/sfForm.class.php#L48